elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/framework/SQLUnitTest.java

```diff
@@ -348,7 +348,7 @@ public static void init(String sqlDialect) {
             throw new IllegalStateException(e);
         }
 
-        metaDataStore = new MetaDataStore(ClassScanner.getAllClasses("com.yahoo.elide.datastores.aggregation.example"));
+        metaDataStore = new MetaDataStore(ClassScanner.getAllClasses("com.yahoo.elide.datastores.aggregation.example"), true);
```

Comment:
Can we set this to false? I think this does not need to be true for the test cases relying on SQLUnitTest to work. If it can work with false, then it will test the other scenario that we can do analytical queries without populating /table, /column etc.

Reply:
If it does not work then we need test cases which verify that we can do analytical queries without populating /table, /column etc.